ipapython/install/common.py

```diff
@@ -63,15 +63,15 @@ def parent(self):
         raise AttributeError('parent')
 
     def _install(self):
-        for _nothing in self._installer(self.parent):
+        for unused in self._installer(self.parent):
```

Comment:
Might be going out on a limb here because the installer code is uberconfusing, but...
at a glance it looks like the only place `_installer` and `_uninstaller` are defined or
referenced is right here in this class, and they both yield a single `None`.

What happens if we delete `_installer` and `_uninstaller` and remove the `for` loop(s)?

Reply:
Yeah, it's so überconfusing that I don't like to touch it. I *think* the functions are called by ``Installable._configure``.

Reply:
Righto, let's leave it alone :)